### PR TITLE
Screen share

### PR DESCRIPTION
**Pull request by Jigar-bhaliya**

### CODE REVIEW

File: bigbluebutton-html5/imports/api/web-list/server/publishers.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const { meetingId, requesterUserId } = extractCredentials(this.userId);
+
+  Logger.debug(`Publishing web List for ${meetingId} ${requesterUserId}`);
+
+  return WebList.find({ meetingId });
+}
+
+function publish(...args) {
+  const boundScreens = webList.bind(this);
+  return boundScreens(...args);
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/api/web-list/server/publishers.js:21]: empty line before return

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabWebContent/TabWebContent.jsx [hunk 2]
```diff
@@ -1,27 +1,72 @@
...
+const TabWebContent = ({
+  webUrl,
+  selectedItem,
+  onAddWebClick,
+  onChangeWebUrl,
+  onSelectedUrl,
+  webList,
+  renderUrlError,
+  addDisabled,
+
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/Tabs/components/TabWebContent/TabWebContent.jsx]: remove empty line

File: bigbluebutton-html5/imports/ui/components/Tabs/service.js [hunk 3]
```diff
@@ -26,6 +27,17 @@ const takePresenterRole = () => makeCall('assignPresenter', Auth.userID);
 
 const getVideoList = () => VideoList.find({ meetingId: Auth.meetingID }).fetch();
 
+const isWebUrl = (str) => {
+  const regexp = /^(?:(?:https?|ftp):\/\/)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,})))(?::\d{2,5})?(?:\/\S*)?$/;
+  if (regexp.test(str)) {
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/Tabs/service.js:33]: empty line before if

File: bigbluebutton-html5/imports/ui/components/app/AppView.jsx [hunk 4]
```diff
@@ -14,32 +14,24 @@ const AppView = ({
...
+        <TabsContainer
+          {...props}
+          handleWhiteboardClick={handleWhiteboardClick}
+          whiteboardOverlay={whiteboardOverlay}
+          isMenuOpen={isMenuOpen}
+          onMenuToggle={onMenuToggle}
+        />
+        )
+      }
+      <section className={cx('main-container py-2 px-5 flex items-center justify-between flex-col relative', { 'w-11/12': isMenuOpen, 'w-full': !isMenuOpen })}>
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/app/AppView.jsx]: multi-line

File: bigbluebutton-html5/imports/ui/components/external-web-frame/ExternalWebContainer.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import ExternalWebView from './ExternalWebView';
+
+const ExternalWebContainer = props => (
+  <ExternalWebView {...{ ...props }} />
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/external-web-frame/ExternalWebContainer.jsx]: `<ExternalWebView { ...props } />`

File: bigbluebutton-html5/imports/ui/components/external-web-wrapper/styles.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  bottom: 0;
+}
+
+.webWrapper{
+  position: absolute;
+  top: 60px;
+  bottom: 60px;
+  left: 0;
+  right: 0;
+}
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/external-web-wrapper/styles.scss]: add empty line at end of file

File: bigbluebutton-html5/imports/api/web-list/server/publishers.js (hunk 1 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/api/web-list/server/publishers.js:21]: added

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabWebContent/TabWebContent.jsx (hunk 2 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/Tabs/components/TabWebContent/TabWebContent.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/Tabs/service.js (hunk 3 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/Tabs/service.js:33]: added

File: bigbluebutton-html5/imports/ui/components/app/AppView.jsx (hunk 4 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/app/AppView.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/external-web-frame/ExternalWebContainer.jsx (hunk 5 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/external-web-frame/ExternalWebContainer.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/external-web-wrapper/styles.scss (hunk 6 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/external-web-wrapper/styles.scss]: added

**Pull request merged**